lib/p2p/Peer.ts

```diff
@@ -124,8 +124,8 @@ class Peer extends EventEmitter {
 
   public get label(): string {
     return this.nodePubKey ||
-      this.expectedNodePubKey ? `${this.expectedNodePubKey}@${addressUtils.toString(this.address)}` :
-      addressUtils.toString(this.address);
+      (this.expectedNodePubKey ? `${this.expectedNodePubKey}@${addressUtils.toString(this.address)}` :
```

Comment:
A small style nit:
```
      (this.expectedNodePubKey
        ? `${this.expectedNodePubKey}@${addressUtils.toString(this.address)}`
        : addressUtils.toString(this.address));
```
I find this more readable when we separate the if/else clauses on new lines.